lib/web_ui/lib/src/engine/canvaskit/image.dart

```diff
@@ -376,7 +376,8 @@ class CkImage implements ui.Image, StackTraceDebugger {
     ui.ImageByteFormat format = ui.ImageByteFormat.rawRgba,
   }) {
     assert(_debugCheckIsNotDisposed());
-    if (videoFrame != null) {
+    // Web codecs do not support I420, I444, I422 videoFrame formats.
```

Comment:
WebCodecs do support these formats. It's just that `readPixelsFromVideoFrame` doesn't convert those to RGBA.

Reply:
right. Updated.